hasura/metadata/databases/default/tables/public_distributions.yaml

```diff
@@ -5,6 +5,9 @@ object_relationships:
 - name: epoch
   using:
     foreign_key_constraint_on: epoch_id
+- name: profile
```

Comment:
I'm just naming this profile as it matches convention... but we could name it something else